.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     parameters:
       stack:
         type: enum
-        enum: ["heroku-16", "heroku-18", "heroku-20"]
+        enum: ["heroku-18", "heroku-20"]
```

Comment:
Nitpick: Shouldn't we change heroku stack here too?

Reply:
This is CircleCI configuration and we don't use CIrcleCI. This is the upstream CI. We don't really care. We trust upstream that the CI is good. No CI on our side.